### PR TITLE
Add references to RTP taxonomy draft

### PR DESCRIPTION
**Pull request by murillo128**

### CODE REVIEW

**Comment by murillo128** [comment]
@youennf PTAL

File: codec-agnostic-rtp-payload-format.md [hunk 1]
```diff
@@ -54,6 +55,69 @@ Introduction
 
 The objective of this spec is to create a generic RTP packetization format that can be used with any video codec or encrypted content and that allows SFUs to perform layer selection without requiring access to the codec payload.
 
+Media paquetization and depaquetization
```
Comment (youennf) [inline codec-agnostic-rtp-payload-format.md]:
```suggestion
Media packetization and depacketization
```

File: codec-agnostic-rtp-payload-format.md [hunk 2]
```diff
@@ -54,6 +55,69 @@ Introduction
...
+=======================
+
+As per {{RFC7656}} the generic packetizer will define a Media Packetizer allows to transfor a single Encoded Stream into one or several RTP packets
```
Comment (youennf) [inline codec-agnostic-rtp-payload-format.md]:
```suggestion
As per {{RFC7656}} the generic packetizer will define a Media Packetizer that transforms a single Encoded Stream into one or several RTP packets.
```

File: codec-agnostic-rtp-payload-format.md [hunk 3]
```diff
@@ -54,6 +55,69 @@ Introduction
 
 The objective of this spec is to create a generic RTP packetization format that can be used with any video codec or encrypted content and that allows SFUs to perform layer selection without requiring access to the codec payload.
 
+Media paquetization and depaquetization
+=======================
+
+As per {{RFC7656}} the generic packetizer will define a Media Packetizer allows to transfor a single Encoded Stream into one or several RTP packets  
+ which can be therefore be processed by a Media Depacketizer to reconstruct the Encoded Stream.
```
Comment (youennf) [inline codec-agnostic-rtp-payload-format.md]:
```suggestion
These RTP packets are sent over the wire to a Media Depacketizer that will reconstruct the Encoded Stream.
```

File: codec-agnostic-rtp-payload-format.md [hunk 4]
```diff
@@ -54,6 +55,69 @@ Introduction
...
+           |  RTP-Based Security  |   |  RTP-Based Security  |
+           +----------------------+   +----------------------+
+                      |                          |
+              Secured RTP Stream   Secured Redundancy RTP Stream
+                      V                          V
+           +----------------------+   +----------------------+
+           |   Media Transport    |   |   Media Transport    |
+           +----------------------+   +----------------------+
+
+             Figure 1: Sender Side Concepts in the Media Chain with Encryption
```
Comment (youennf) [inline codec-agnostic-rtp-payload-format.md]:
```suggestion
             Figure 1: Sender Side Concepts in the Media Chain with Application-level Media Encryption 
```

File: codec-agnostic-rtp-payload-format.md [hunk 5]
```diff
@@ -54,6 +55,69 @@ Introduction
...
+           |   Media Transport    |   |   Media Transport    |
+           +----------------------+   +----------------------+
+
+             Figure 1: Sender Side Concepts in the Media Chain with Encryption 
+             
+```
+ 
+This generic packetization does not change how the mapping between one or several encoded or dependant streams are mapped to the RTP streams or how the synchronization sources(s) (SSRC) are assigned. 
+
+The generic packetizer only supports SRST modes when Scalale Video Coding (SVC) is in use.
```
Comment (youennf) [inline codec-agnostic-rtp-payload-format.md]:
```suggestion
The generic packetizer only supports Single RTP stream on a Single media Transport (SRST) when Scalale Video Coding (SVC) is in use.
```

**Review by youennf — APPROVED** [review]
LGTM. I proposed some editorial changes.
I think it might be worth adding another figure for the current architecture.
It is mostly figure 1 without Media Encryptor and with 'Media Packetizer (codec aware)'
And update figure 1 to mention 'Media Packetizer (generic)'

**Pull request merged**